### ISSUE TITLE
Appears to be broken in Flutter 3.22.0

### ISSUE THREAD

**Issue by HankG** [xsahil03x/multi_trigger_autocomplete#26]
I've created a bug over on the Flutter board related to this: https://github.com/flutter/flutter/issues/148653

The TL;DR is that it appears that something in [commit 8e418d18185697e56024ef3d5784afba4ac96370](https://github.com/flutter/flutter/commit/8e418d18185697e56024ef3d5784afba4ac96370) seems to have broken something that is being done with rendering the popup on Linux. I haven't had luck figuring out what exactly yet though. I'm going to keep dabbling with it to see if there is a work around. For now, using the standard example, in previous releases one would get:
![WorkingExample](https://github.com/xsahil03x/multi_trigger_autocomplete/assets/1683691/593808a0-7e7f-40f8-91a0-153b30c72331)

...but now one gets:

![BrokenExample](https://github.com/xsahil03x/multi_trigger_autocomplete/assets/1683691/2b271fd5-0100-411e-a1cc-99bc230ea42f)

The case degenerates in a different way in my own app though.

**Comment by HankG** [comment]
This change probably fixes it, since this was the test case I reported as being broken. https://github.com/flutter/flutter/issues/148653
